.github/workflows/ci.yml

```diff
@@ -12,3 +12,30 @@ jobs:
 
       - name: Run tests
         run: python -m unittest test_vectors.py
+
+  format-markdown:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Format Markdown with markdownlint
+        run: |
+          npm install -g markdownlint-cli
+          markdownlint --disable MD013 --fix .
+          git add -A
+          git diff --name-only --cached --exit-code
```

Comment:
Why `--name-only`?

Reply:
Oops, removing it now to include the error diff in the output.

Reply:
Already done :wink:.